### DISCUSSION TITLE
[Showcase] Show your custom config of uosc

### DISCUSSION THREAD

**Discussion by aicynide** [tomasklaen/uosc#541] [Show and tell]
With Screenshot

**Comment by OmiChrome** [comment]
_**Minimal  | Usefull**_
- **Showcase**
![image](https://user-images.githubusercontent.com/108952834/235245655-0835c13b-172e-4a67-b429-c327dcc5ae9c.png)


https://user-images.githubusercontent.com/108952834/235249315-f756df55-e958-4514-8427-d583044b95b2.mp4




**[Dotfiles](https://github.com/omaru-ok/Pwsh-env/blob/main/Personal/portable_config.7z)**

* Inspired from 
* * hook007 [mpv_Lazy](https://github.com/hooke007/MPV_lazy)
* * dyphire [mpv-config](https://github.com/dyphire/mpv-config)
* * natural-harmonia-gropius [mpv-config](https://github.com/natural-harmonia-gropius/mpv-config)

**Reply by aicynide** [reply]
Awesome!
